tests/integration/cattletest/core/test_authorization.py

```diff
@@ -1,6 +1,23 @@
 from common_fixtures import *  # NOQA
 
 
+def test_client_access(clients):
+    typesLen = {
+        'admin': 89,
+        'agent': 8,
+        'user': 68,
+        'agentRegister': 4,
+        'test': 140,
+        'readAdmin': 89,
+        'token': 2,
+        'superadmin': 140,
+        'service': 89,
+        'project': 68,
+    }
+    for tuple in clients.items():
+        assert typesLen[tuple[0]] == len(tuple[1].schema.types.items())
```

Comment:
Hmm.  I forget what drone spits out when the test fails.  Does it spit out which client is mismatching?  I'm just anticipating TFs occurring when people add new resources.